en/power_module/cuav_can_pmu.md

```diff
@@ -0,0 +1,79 @@
+# CAUV CAN PMU
```

Comment:
@cuhome For both boards, might be good to include a what's in the box screenshot or list?